transport/spi/src/main/java/org/kaazing/gateway/transport/LoggingFilter.java

```diff
@@ -134,14 +135,8 @@ public static boolean addIfNeeded(Logger logger, IoSession session, String trans
         if (!logger.isInfoEnabled()) {
             return false;
         }
-        String user = getUserIdentifier(session);
         String loggingFilterName = transportName + "#logging";
```

Comment:
I extracted the logic for getting the user into the `getLoggingFormat` method in order to be able to also use this in derived classes of the LoggingFilter class.